php/class-wp-cli.php

```diff
@@ -858,36 +864,39 @@ private function create_guest_author( $author ) {
 			$guest_author = $coauthors_plus->guest_authors->get_guest_author_by( 'user_login', $author['user_login'], true );
 		}
 
-		if ( ! $guest_author ) {
-			WP_CLI::line( '-- Not found; creating profile.' );
+		if ( $guest_author ) {
+			// translators: Guest Author ID.
+			return WP_CLI::warning( sprintf( esc_html__( '-- Author already exists (ID #%s); skipping.', 'co-authors-plus' ), $guest_author->ID ) );
+		}
 
-			$guest_author_id = $coauthors_plus->guest_authors->create(
-				array(
-					'display_name' => $author['display_name'],
-					'user_login'   => $author['user_login'],
-					'user_email'   => $author['user_email'],
-					'first_name'   => $author['first_name'],
-					'last_name'    => $author['last_name'],
-					'website'      => $author['website'],
-					'description'  => $author['description'],
-					'avatar'       => $author['avatar'],
-				)
-			);
+		WP_CLI::line( esc_html__( '-- Not found; creating profile.', 'co-authors-plus' ) );
 
-			if ( $guest_author_id ) {
-				WP_CLI::line( sprintf( '-- Created as guest author #%s', $guest_author_id ) );
+		$guest_author_id = $coauthors_plus->guest_authors->create(
+			array(
+				'display_name' => $author['display_name'],
+				'user_login'   => $author['user_login'],
+				'user_email'   => $author['user_email'],
+				'first_name'   => $author['first_name'],
+				'last_name'    => $author['last_name'],
+				'website'      => $author['website'],
+				'description'  => $author['description'],
+				'avatar'       => $author['avatar'],
+			)
+		);
 
-				if ( isset( $author['author_id'] ) ) {
-					update_post_meta( $guest_author_id, '_original_author_id', $author['ID'] );
-				}
+		if ( is_wp_error( $guest_author_id ) ) {
+			// translators: The error message.
+			return WP_CLI::warning( sprintf( esc_html__( '-- Failed to create guest author: %s', 'co-authors-plus' ), $guest_author_id->get_error_message() ) );
+		}
 
-				update_post_meta( $guest_author_id, '_original_author_login', $author['user_login'] );
-			} else {
-				WP_CLI::warning( '-- Failed to create guest author.' );
-			}
-		} else {
-			WP_CLI::line( sprintf( '-- Author already exists (ID #%s); skipping.', $guest_author->ID ) );
+		// translators: Guest Author ID.
+		WP_CLI::success( sprintf( esc_html__( '-- Created as guest author #%s', 'co-authors-plus' ), $guest_author_id ) );
```

Comment:
Nitpick, but I think it's better if we have the success output message as the last thing, after the update_post_meta calls, so we are sure we only output a success after everything was done

Reply:
That's a good point @leogermani I updated the code here: https://github.com/Automattic/Co-Authors-Plus/pull/879/commits/105d38b92079662ed5fce28d744c616dc345a7d0#diff-a52979a3ff3a75e65a5166f5fc4a259ef7e243368b5953e587054bbfbada9865R899